rendermodules/montage/run_montage_job_for_section.py

```diff
@@ -167,8 +102,23 @@ def __init__(self, schema_type=None, *args, **kwargs):
         # Assigning the solver_executable to a different variable and removing it from args
         self.solver_executable = self.args['solver_executable']
         self.args.pop('solver_executable', None)
+        self.clone_section_stack = self.args['clone_section_stack']
+        self.args.pop('clone_section_stack')
+        if self.clone_section_stack:
+            if (self.args['first_section']!=self.args['last_section']):
+                raise ValidationError("first section and last section not equal")
```

Comment:
Is there a reason why we shouldn't let this work for multiple zs?  It could be helpful in manual/batch processing and shouldn't be a very different implementation.

Reply:
i can't see a reason... i've changed the module to allow it